gobblin-api/src/main/java/org/apache/gobblin/configuration/ConfigurationKeys.java

```diff
@@ -327,6 +327,10 @@
   public static final String CONVERTER_SKIP_FAILED_RECORD = "converter.skipFailedRecord";
   public static final String CONVERTER_AVRO_SCHEMA_KEY = "converter.avroSchema";
   public static final String CONVERTER_IGNORE_FIELDS = "converter.ignoreFields";
+  public static final String CONVERTER_INPUT_TIMEZONE = "converter.input.timezone";
```

Comment:
This should be part of the converter class since its specific to that date converter